lib/views/pages/welcome.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:sun_flutter_capstone/utils/routing.dart';
+
+class WelcomePage extends StatelessWidget {
+  const WelcomePage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Column(
+        children: [
+          ElevatedButton(
+              onPressed: () => {navigateTo(context, '/login')},
+              child: const Text('Login')),
+          ElevatedButton(
+              onPressed: () => {navigateTo(context, '/register')},
+              child: const Text('Register'))
+        ],
```

Comment:
If you add commas at the end, it will properly format like this 🙏 
```suggestion
        children: [
          ElevatedButton(
            onPressed: () => {navigateTo(context, '/login')},
            child: const Text('Login'),
          ),
          ElevatedButton(
            onPressed: () => {navigateTo(context, '/register')},
            child: const Text('Register'),
          ),
        ],
```

Reply:
will add commas to it, miss 🏃‍♀️